core/src/main/java/com/datastax/oss/driver/internal/core/os/Native.java

```diff
@@ -150,4 +178,27 @@ private Timeval(Runtime runtime) {
       GET_PID_AVAILABLE = getPidAvailable;
     }
   }
+
+  private static class PlatformLoader {
+
+    private static final Platform PLATFORM;
+
+    static {
+      Platform platform;
+      try {
+        Class<?> platformClass = Class.forName("jnr.ffi.Platform");
+        Method getNativePlatform = platformClass.getMethod("getNativePlatform");
+        platform = (Platform) getNativePlatform.invoke(null);
+      } catch (Throwable t) {
+        platform = null;
+        if (LOG.isDebugEnabled())
```

Comment:
I see that contrary to 3.x, in 4.x we only log at DEBUG level, again I think we should harmonize the code.

Reply:
done